app/routes/send-endringsmelding.tsx

```diff
@@ -1,19 +1,113 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
 import { ESanitySteg } from '~/typer/sanity/sanity';
 import { TypografiTyper } from '~/typer/typografi';
-import { useTekster } from '~/hooks/contextHooks';
+import { useSpråk, useTekster } from '~/hooks/contextHooks';
 import InnholdKonteiner from '~/komponenter/innholdkonteiner/InnholdKonteiner';
+import css from './send-endringsmelding.module.css';
+import cssFritekst from './fritekstfelt.module.css';
+
+import { Button, Textarea } from '@navikt/ds-react';
+import { hentPathForSteg } from '~/utils/hentPathForSteg';
+import { useNavigate } from '@remix-run/react';
 
 export default function SendEndringsmelding() {
   const tekster = useTekster(ESanitySteg.SEND_ENDRINGER);
+  const navigate = useNavigate();
+  const [språk] = useSpråk();
+
+  const spesialTegnRegex = /[!@#$%^&*()?"{}|<>+¨=]/;
+  const MAKS_INPUT_LENGDE = 1000;
+
+  const i18nInnhold = {
+    counterTooMuch: hentI18nInnhold(true),
+    counterLeft: hentI18nInnhold(false),
+  };
+
+  function hentI18nInnhold(tegnIgjen: boolean) {
+    switch (språk) {
+      case 'nb':
+        return tegnIgjen ? 'tegn igjen' : 'tegn for mye';
+      case 'nn':
+        return tegnIgjen ? 'teikn igjen' : 'teikn for mykje';
+      case 'en':
+        return tegnIgjen ? 'characters left' : 'characters too many';
+    }
+  }
+
+  const [tekstInputOK, settTekstInputOK] = useState<boolean>(false);
+  const [manglerTekst, settManglerTekst] = useState<boolean>(false);
+  const [brukerSpesialtegn, settBrukerSpesialtegn] = useState<boolean>(false);
+  const [minimumTegnOppfylt, settMinimumTegnOppfylt] = useState<boolean>(false);
+  const [knappTrykketPå, settKnappTrykketPå] = useState<boolean>(false);
+
+  const sjekkTekst = (innholdVerdi: string) => {
```

Comment:
```suggestion
  const validerTekst = (innholdVerdi: string) => {
```
Eventuelt `validerTekstInnhold`.
Er valider et mer presist ord enn sjekk?

Reply:
ja, endret det her https://github.com/bekk/nav-familie-endringsmelding/pull/34/commits/3f8c578edbf13803c5466821e4d7c309518ea13f